src/services/study/network.js

```diff
@@ -138,16 +139,21 @@ export function fetchNetworkElementsInfos(
     substationsIds,
     elementType,
     infoType,
-    inUpstreamBuiltParentNode
+    inUpstreamBuiltParentNode,
+    nominalVoltages = null
```

Comment:
undefined ?

Reply:
see `fetchEquipmentsIds` in network-map.js

Reply:
see #2141

Reply:
Done